CIP-0062/README.md

```diff
@@ -0,0 +1,443 @@
+
+### VotingCredentials
+```ts
+interface VotingCredentials {
+  votingKey: PublicKey
+  stakingCredential: PublicKey
+}
+```
+Information used to represent a wallet's voting credentials.
+
+* votingKey - Derivation as described within [CIP-0036](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036), wallets should use `address_index`= 0.
+* stakingCredential - At the moment, the only supported staking credential is a public staking key. This is used to represent the wallet's staked ADA.
+
+### BlockDate
+
+```ts
+interface BlockDate {
+  epoch: number
+  slot: number
+}
+```
+
+* epoch - an epoch value on the voting block chain.
+* slot - a block slot number on the voting block chain.
+
+#### Proposal
+
+```ts
+interface Proposal {
+  votePlanId: string
+  voteOptions: number[]
+  votePublic: boolean
+  proposalIndex: number
+}
+```
+
+Proposal information.
+
+* votePlanId - Hex encoded string to represent vote plan unique identifier. This will be the same as anchored on voting chain, and is managed by the dApp.
+* voteOptions - List of possible "choices" which can be made for this proposal.
+* votePublic - Boolean indicating whether the vote is public or private.
+
+### Vote
+
+```ts
+interface Vote {
+  proposal: Proposal
+  choice: number
+  expiration: BlockDate
+  purpose: VotingPurpose
+  spendingCounter: number
+}
+```
+
+An individual raw Unsigned Vote record.
+
+* proposal - The [proposal](#proposal) being voted on.
+* choice - This value MUST match one of the available `voteOptions` in the `proposal` element. An `UnknownChoiceError` should be thrown if the value is not one of the valid `voteOptions` of the `proposal`.
+* expiration - Voting Chain epoch \& slot for when the vote will expire. This value is supplied and maintained by the dApp, and forms a necessary component of a [Vote](#vote)
+* purpose - The [voting purpose](#voting-purpose) being voted on. (Currently always 0).
+* spendingCounter - The spending counter is used to prevent double voting. The spending counter for the vote transaction will be supplied and maintained by the dApp. The dApp will manage supplying this in the correct order, and this should not be enforced by the Wallet.
+
+It is required to attach it to the vote according to [Jormungandr Voting] (<https://input-output-hk.github.io/jormungandr/jcli/vote.html#voting>).
+
+## Delegation
+
+```ts
+interface Delegation {
+    delegations: GovernanceKey[]
+    purpose: VotingPurpose
+}
+```
+
+The record of a voters delegation.
+
+* delegations - List of [Governance Keys](#governancekey) and their voting weight to delegate voting power to.
+* purpose - The [Voting Purpose](#voting-purpose) being delegated.  A voter may have multiple active delegations for different purposes, but only 1 active delegation per unique Voting Purpose.
+
+## DelegatedCertificate
+
+```ts
+interface DelegatedCertificate {
+  delegations: GovernanceKey[]
+  stakingPub: string
+  rewardAddress: string
```

Comment:
I guess this should be renamed to match the latest CIP-36 specs: https://github.com/cardano-foundation/CIPs/pull/373

```suggestion
  paymentAddress: string
```

Reply:
@refi93 

Yes good spot - I was going to reflect that change here once #373 is merged. 

Next week I will do a pre-emptive PR to this CIP to match the changes in #373 and #427 so when/if they are merged this CIP is aligned.